app/migration/school_period_extractor.rb

```diff
@@ -40,7 +42,11 @@ def build_school_periods_from_induction_records
                                                      start_date: induction_record.start_date,
                                                      end_date: induction_record.end_date,
                                                      start_source_id: induction_record.id,
-                                                     end_source_id: induction_record.id)
+                                                     end_source_id: induction_record.id,
+                                                     appropriate_body_type: induction_record.appropriate_body.body_type,
+                                                     programme_type:,
+                                                     appropriate_body_id: induction_record.appropriate_body_id,
+                                                     lead_provider_id: nil)
```

Comment:
I think we need to dig more here otherwise we'll get odd failures.
 - if the AB is not set on the induction record we need to go look at the `induction_programme.school_cohort.appropriate_body` and use that if it is present
 - can `lead_provider_id` only be `nil` if `programme_type` is `school_led`?